### PR TITLE
feat: Add Galaxy conf files for BlueBanquise Core

### PR DESCRIPTION
**Pull request by btravouillon**
Relates to #702.

This should be considered as a preview feature, unsupported method of installation.

### CODE REVIEW

File: roles/advanced_core/galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+namespace: bluebanquise
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: advanced_core
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.6.0
```
Comment (btravouillon) [inline roles/advanced_core/galaxy.yml:7]: For merge into `master` that could be `1.6.99`.

**Comment by oxedions** [comment]
Dear @btravouillon 

Thanks a lot for this PR ! 😊 

I see that the split of roles folders is an issue here.

I propose to remove that in V2:

* /bluebanquise/roles/dhcp_server
* /bluebanquise/roles/advanced_dhcp_server
* ...

In the meantime, your dev is the solution.

**Comment by oxedions** [comment]
Hello @btravouillon 
So I added a collection galaxy.yml file for each collections, trying to gather each times authors written in Readme files.
Is it OK for you this way and usable ? 🙂

**Comment by oxedions** [comment]
Closing as discussed P2P 🙂

**Pull request closed without merging**